rep-2008.rst

```diff
@@ -0,0 +1,777 @@
+- `compute substrate-agnostic`: benchmarks should be able to run on different hardware acceleration technology solutions. For that purpose, a CPU-centric framework (as opposed to an acceleration technology-specific framework) for benchmarking and/or tracing is the ideal choice.
+- `automated`: benchmarks and related source code should be designed with automation in mind. Once ready, creating a benchmark and producing results should be (ideally) a fully automated process.
+- `hardware farm mindset`: benchmarks will be conducted on hardware embedded platforms sitting in a farm-like environment (redundancy of tests, multiple SoCs/boards) with the intent of validating and comparing different technologies.
+
+Accounting for all of this, in this REP, we adopt a grey-box and non-functional benchmarking approach for hardware acceleration that allows to evaluate the relative performance of accelerated ROS 2 individual nodes and complete computational graphs. To realize it in a technology agnostic-manner, we select the Linux Tracing Toolkit next generation (`LTTng <https://lttng.org/>`_) which will be used for tracing and benchmarking. 
+
+Differences between tracing and benchmarking
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Tracing and benchmarking can be defined as follows:
+
+- `tracing`: a technique used to understand what goes on in a running software system.
+- `benchmarking`: a method of comparing the performance of various systems by running a common test.
+
+From these definitions, inherently one can determine that both benchmarking and tracing are connected in the sense that the test/benchmark will use a series of measurements for comparison. These measurements will come from tracing probes. In other words, tracing will collect data that will then be fed into a benchmark program for comparison.
+
+
+.. _acceleration examples:
+
+Acceleration examples
+^^^^^^^^^^^^^^^^^^^^^
+
+For the sake of illustrating maintainers and consumers how to build their own acceleration kernels and guarantee interoperability across technologies, a ROS 2 meta-package named `acceleration_examples <https://github.com/ros-acceleration/acceleration_examples>`_ will be maintained and made available. This meta-package will contain various packages with simple common acceleration examples. Each one of these examples should support all hardware acceleration solutions complying with this REP.
+
+In turn, a CI system will be set to build periodically and for every commit the meta-package.
+
+.. _capabilities:
+
+Capabilities
+^^^^^^^^^^^^
+
+The following list describes the hardware acceleration capabilities that hardware acceleration vendors must consider when connecting their firmware and technology solutions to the ROS 2 ecosystem.
+
```

Comment:
Github is having a bad time rendering this section in the preview. I tested it out in another rst renderer and it turned out just fine so I'm guessing it's just a bug.
![image](https://user-images.githubusercontent.com/11024792/135360352-0144a84a-6cb4-4bff-a8e9-16a4ac7f5c23.png)

Reply:
Thanks for looking at this. 

I only iterated locally where I get a nice rendering using `rep2html.py` script. I can confirm it renders HTML fine for me as well.